graph-commons/src/test/scala/ch/datascience/generators/Generators.scala

```diff
@@ -165,7 +166,8 @@ object Generators {
                }
     port <- httpPorts
     host <- nonEmptyStrings()
-  } yield s"$protocol://$host:$port"
+    path <- relativePaths(maxSegments = 2)
```

Comment:
I guess this is only used to test that absolute URLs cannot be paths? Otherwise `maxSegments` should probably not be `2`.

Reply:
This is a generator for property-based tests. I added some path for the URLs being generated here thinking that tests which depend on URLs check what happens if there's some path.